app/controllers/api/v1/rollbacks_controller.rb

```diff
@@ -23,6 +26,22 @@ def create
       def rollback_params
         params.require(:rollback).permit(:date, :keep, :reason, :user_id)
       end
+
+      def collection
+        @collection ||= Rollback.descending.paginate(page, PER_PAGE)
+      end
+
+      def paginate
+        response.headers["X-Pagination"] = {
```

Comment:
I chose this approach as I didn't want to modify the json response. ATM we are not using root elements in responses:
`{rollbacks:[{},{}...]}`

but just arrays with the records:
`[{},{}...]`